@types/APIClients.ts

```diff
@@ -58,6 +58,13 @@ interface APIClientInterface {
     invalidate(): Promise<void>;
 }
 
+type CertificateConfiguration = {
```

Comment:
This is a style preference.  I normally sort everything alphabetically as this makes it more developer-friendly and it is easy on the eyes for when you need to debug and deep dive into codes.  If you are looking for `clientCertificate`, you need to traverse 3 lines to get to it.